test/integration/scripts/run-tests.sh

```diff
@@ -23,16 +23,32 @@ while [[ $# -gt 0 ]]; do
   esac
 done
 
+# setup hosts
+addHost() {
+  local hostname="$1"
+  
+  # check first to avoid sudo prompt if host is already set
+  if hostile list | grep -q "127.0.0.1 $hostname"; then
+    echo "$hostname already set"
+  else
+    sudo hostile set 127.0.0.1 "$hostname"
+  fi
+}
```

Comment:
If hosts aren't present you will be prompted for your password so it can write to /etc/hosts. If hosts are present, it will not prompt you. In CI, there is a step before running tests that add them so we dont need to navigate sudo here in ci.

Note that we didnt need sudo-prompt here. Unless I misunderstood the original suggestion, its just not necessary for this use-case.

Reply:
> Note that we didnt need sudo-prompt here. 

So why do we include it?

Reply:
> Note that we didnt need sudo-prompt here.

So why do we include it?

Reply:
I'm referring to this npm package https://www.npmjs.com/package/sudo-prompt, not the use of `sudo` here.